client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,10 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
+        <Link to='/'>
```

Comment:
Nice. You could also have used this.props.history.push('/') if you had used render props to pass the history object down to this component.